lib/ExpandablePicker/ExpandablePickerAccessibilitySupport.js

```diff
@@ -10,18 +10,14 @@ module.exports = {
 	/**
 	* @private
 	*/
-	initAccessibility: kind.inherit(function (sup) {
-		return function (props) {
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (was, is, prop) {
+			var enabled = !this.accessibilityDisabled;
 			sup.apply(this, arguments);
-			this.$.currentValue.observe('content', this.changeLabel, this);
-			this.changeLabel();
+			this.$.headerWrapper.setAttribute('aria-labelledby', enabled ? this.$.headerWrapper.getId() : null);
```

Comment:
Guessing you'll have the same labelledby issue here since $.header contains the text but will be `aria-hidden` and therefore not read when its ancestor is set as the labelled by id.